### PR TITLE
fix: design guild

### PR DESCRIPTION
**Pull request by BOBOtoTHEmoon**

### CODE REVIEW

File: components/about-us/sub-pages/guilds/guild.tsx [hunk 1]
```diff
@@ -58,9 +58,9 @@ function Guild() {
           />
           <Section
             title='Design Guild'
-            text='The DAOlationships Guild facilitates and coordinates relationships between external organizations and
-          the guilds within BanklessDAO.
-          This guild helps us best represent our DAO to anyone outside bDAO.'
+            text='Clear communication of the DAO mission through intelligent next-gen design. We establish powerful symbols, build a
+                  narrative through visual aid, generate funding through art, always using integrity in our images and
+                  content, and as a culture, we reward action and embrace risk.'
```
Comment (MantisClone) [inline components/about-us/sub-pages/guilds/guild.tsx:63]:
```suggestion
                  content. As a culture, we reward action and embrace risk.'
```

File: components/about-us/sub-pages/guilds/guild.tsx [hunk 2]
```diff
@@ -58,9 +58,9 @@ function Guild() {
           />
           <Section
             title='Design Guild'
-            text='The DAOlationships Guild facilitates and coordinates relationships between external organizations and
-          the guilds within BanklessDAO.
-          This guild helps us best represent our DAO to anyone outside bDAO.'
+            text='Clear communication of the DAO mission through intelligent next-gen design. We establish powerful symbols, build a
+                  narrative through visual aid, generate funding through art, always using integrity in our images and
```
Comment (MantisClone) [inline components/about-us/sub-pages/guilds/guild.tsx:62]:
```suggestion
                  narrative through visual aid, and generate funding through art. We use integrity in our images and
```

**Review by MantisClone — CHANGES_REQUESTED** [review]
@BOBOtoTHEmoon Thanks for this PR.  I realized the text we provided you was not grammatically correct so I added some minor changes.

**Comment by BOBOtoTHEmoon** [comment]
oh, thank you ser

**Review by MantisClone — APPROVED** [review]

**Pull request merged**